src/components/Scorecards/ScorecardsServicePage/ScorecardsServicePage.tsx

```diff
@@ -57,6 +58,8 @@ export const ScorecardsServicePage = () => {
     ScorecardServiceScoresRule[]
   >([]);
 
+  const cortexBaseUrl = config.getOptionalConfig('cortex.frontend.baseUrl');
```

Comment:
Looks like this needs to be
```suggestion
  const cortexBaseUrl = config.getOptionalString('cortex.frontend.baseUrl');
```
to read the string value instead of a `Config` object.